bwc-test/src/test/java/SecurityBackwardsCompatibilityIT.java

```diff
@@ -35,6 +62,11 @@ private void testSetup() {
         CLUSTER_NAME = System.getProperty("tests.clustername");
     }
 
+    @Override
+    protected final boolean preserveClusterUponCompletion() {
```

Comment:
Just for my knowledge what is this boolean for?

Reply:
Its to bypass this [method](https://github.com/opensearch-project/OpenSearch/blob/main/test/framework/src/main/java/org/opensearch/test/rest/OpenSearchRestTestCase.java#L364) which throws a `WarningFailureException` when using snapshots. It fails because its trying to json parse a response along with a warning that's printed out around the dot syntax for system index deprecation.